crates/rpc/rpc-engine-api/src/engine_api.rs

```diff
@@ -268,8 +268,6 @@ where
             .validator
             .validate_version_specific_fields(EngineApiMessageVersion::V4, payload_or_attrs)?;
 
-        validate_execution_requests(&execution_requests)?;
```

Comment:
we don't remove this, we call `self.validator.validate_execution_requests(&execution_requests)?;`